src/__tests__/lib/rules/prefer-in-document.js

```diff
@@ -135,18 +187,6 @@ const invalid = [
     `expect(screen.getAllByRole('foo')).toHaveLength(1,2,3,)`,
     `expect(screen.getByRole('foo')).toBeInTheDocument()`
   ),
-  invalidCase(
-    `expect(screen.getAllByRole('foo')).toHaveLength(0//comment
-)`,
-    `expect(screen.getByRole('foo')).not.toBeInTheDocument(//comment
-)`
-  ),
-  invalidCase(
-    `expect(screen.getAllByRole('foo')).toHaveLength(1,//comment
-)`,
-    `expect(screen.getByRole('foo')).toBeInTheDocument(//comment
-)`
-  ),
```

Comment:
Since now `expect(screen.getAllBy*('')).toHaveLength(0 or 1)` is a valid case and has been added to `valid` array, I removed these two cases.

Reply:
nit: generally in these situations I like to include these cases in `valid` for explicitly